### ISSUE TITLE
missing sqlite db

### ISSUE THREAD

**Issue by stimpy2318** [Kunde21/raspberry-dev#1]
I recently started using your gitea docker image from Docker hub and wanted to let you know that I ran into a little issue when my pi had to be shutdown. The sqlite database file doesn't appear to be in the volume location on the host system. I see in the app.ini file (since that and the git repo and other gitea config exists in the volume) that it should be located in data/gitea.db but I can't find it in the volume. This resulted in gitea acting like a new install after restart. Fortunately I had only been running it for a few days and can recover the git data easily enough, but it will be a headache to recreate the gitea configuration and redo any messed up ssh keys.

I realize that this isn't exactly a production system, but I thought I would let you know in case you wanted to update the image or possibly share the updated image build process so that people who are using this can correct the problem. Especially since this image is one of the top downloaded gitea images on Dockerhub.

**Comment by Kunde21** [comment]
Do you have the version (docker tag) that you were using?  I have some time to recreate and debug, so I'd like to get the bug fixed.

**Comment by stimpy2318** [comment]
`docker pull kunde21/gitea-arm` run a few days ago (on April 6th I think) resulted in an image with checksum of "sha256:87c5612ff13601ca60b29f307ccc35e0c5311c61cc6eb53077e2a7eea329025f"
I am somewhat new to docker and I haven't found a way to determine what tags my particular container has.

**Comment by sonerturkkan** [comment]
I had similar issue. after some little search discover that all the paths have to start "/data", otherwise installer create the locations inside of the docker volume.

**Comment by Kunde21** [comment]
Previously, the defaults on the configuration page pointed to `/data/*` locations.  I'm going to look into ways to make installation and configuration easier.

**Comment by Kunde21** [comment]
I believe I found the issue, missing dependency that wasn't failing properly on startup.  
 
@stimpy2318 @sonerturkkan  Please try creating a new instance from latest or rc (tags:  `latest`, `v1.7.6`, `v1.7`, `v1`, `v1.8.0-rc3`) and ensure the default configuration points to sqlite db at `/data/gitea/gitea.db`

**Comment by stimpy2318** [comment]
That seems to have worked, I did a quick test by clearing out the old volume then pulling the updated image, starting up the docker container, setting up gitea through the webUI, then stopping the docker container and restarting it, my repo and registered user account were still there. I also tried rebooting the Pi running the container then starting up the container again and it was still functional. I also checked the app.ini file and the path for the gitea.db did appear to point to the proper docker volume location.

Thank you for being so responsive!
-Stimpy
